### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Code1705**
committed the changes , please could you merge my pr?

### CODE REVIEW

**Comment by Code1705** [comment]
just could you merge my PR please , after reviewing.

File: README.md [hunk 1]
```diff
@@ -221,3 +258,48 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+ 
+
+
+<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
+
+
+<h2 align="center">Feedback</h2>
+
+If you have any feedback or suggestions please reach out to Project - Maintainer.  
+* [Lalit Kumar Singh](https://github.com/harshita214)
```
Comment (Lalit3716) [inline README.md:273]:
```suggestion
If you have any feedback or suggestions please reach out to: 
* [Lalit Kumar Singh](https://github.com/Lalit3716) 
```

File: README.md [hunk 2]
```diff
@@ -221,3 +258,48 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
+
+
+<h2 align="center">Feedback</h2>
+
+If you have any feedback or suggestions please reach out to Project - Maintainer.  
+* [Lalit Kumar Singh](https://github.com/harshita214) 
+
+
+Or you can create a  <a href="https://github.com/aman34503/harshita214/issues">issue</a> and mention there , which new features or extension can make this Project more good.
```
Comment (Lalit3716) [inline README.md:276]: Correct the url in `href` here

File: README.md [hunk 3]
```diff
@@ -221,3 +258,48 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+
+ ‼️ For [JWOC'23](https://jwoc.tech/) program Admin and Mentor rights of Vita App are given to [Lalit Kumar Singh](https://github.com/whiteHatpro) ‼️
+
+</h3>
+  
+  
+<div id="Bottom"></div>
+
+
+
```
Comment (Lalit3716) [inline README.md:305]: Remove this section

**Review by Lalit3716 — CHANGES_REQUESTED** [review]
Please do these changes, Thanks!

**Comment by Code1705** [comment]
As per you require changes are made , kindly merge my pr

**Comment by Code1705** [comment]
I have done the changes , please could you merge it with the label of jwoc

File: README.md [hunk 4]
```diff
@@ -221,3 +258,38 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
+
+
+<h2 align="center">Feedback</h2>
+
+If you have any feedback or suggestions please reach out to Project - Maintainer.  
+* [Lalit Kumar Singh](https://github.com/Lalit3716) 
+
+
+Or you can create a  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.
```
Comment (Lalit3716) [inline README.md:276]:
```suggestion
If you have any feedback or suggestions you can create an  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.
```

**Review by Lalit3716 — CHANGES_REQUESTED** [review]
Don't make changes to `index.css` in this pull request as it's not really related to improving documentation.
Also please do the change I have mentioned thanks!

**Comment by Code1705** [comment]
see now is it ok?

File: README.md [hunk 5]
```diff
@@ -267,18 +267,7 @@ To maintain a safe and inclusive space for everyone to learn and grow, contribut
...
-If you have any feedback or suggestions please reach out to Project - Maintainer.  
-* [Lalit Kumar Singh](https://github.com/Lalit3716) 
-
-
-Or you can create a  <a href="https://github.com/Vita-App/Vita/issues">issue</a> and mention there , which new features or extension can make this Project more good.
-
-<!-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------ -->
-
-<br>
-
```
Comment (Lalit3716) [inline README.md]: Don't remove this section just make some changes I told you to make in previous review

**Review by Lalit3716 — CHANGES_REQUESTED** [review]
Please undo all the changes you did in `index.css`

**Comment by Code1705** [comment]
now is it ok?
could you check please?

**Comment by Code1705** [comment]
could you merge my PR?

**Comment by Code1705** [comment]
as per your instructions , changes are made now merge my pr

**Comment by Lalit3716** [comment]
@Code1705 Can you reopen this PR idk but it somehow got closed ?

**Comment by Code1705** [comment]
As you told I opened the new issue as update readme , now you can label it under Jwoc and assign me.

**Pull request closed without merging**